README.md

```diff
@@ -1,2 +1,57 @@
-# webpack-config
-A simple yet powerful webpack boilerplate, which can later use as a starting point in your projects.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Leaderboard
+
+> The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service.
+
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+- Webpack
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps. 
+
+1.- Clone the project directory in your terminal 
```

Comment:
- [OPTIONAL] Please it will be nice to outline explicit steps on how to clone it, as this will be of help to non technical users.